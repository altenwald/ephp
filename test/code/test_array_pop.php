<?php
$stack = array("orange", "banana", "apple", "strawberry");
$fruit = array_pop($stack);
var_dump($fruit, $stack);

