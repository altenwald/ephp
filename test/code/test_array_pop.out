string(10) "strawberry"
array(3) {
  [0]=>
  string(6) "orange"
  [1]=>
  string(6) "banana"
  [2]=>
  string(5) "apple"
}
